common/djangoapps/student/migrations/0047_auto__add_field_linkedinaddtoprofileconfiguration_company_identifier__.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'LinkedInAddToProfileConfiguration.company_identifier'
+        db.add_column('student_linkedinaddtoprofileconfiguration', 'company_identifier',
+                      self.gf('django.db.models.fields.TextField')(default='', blank=True),
+                      keep_default=False)
+
+        # Adding field 'LinkedInAddToProfileConfiguration.trk_partner_name'
+        db.add_column('student_linkedinaddtoprofileconfiguration', 'trk_partner_name',
+                      self.gf('django.db.models.fields.CharField')(default='', max_length=10, blank=True),
+                      keep_default=False)
```

Comment:
Yes. If the solutions migration didn't include your fields in their `models` dictionary, South wouldn't know that they'd already been migrated.